walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/openid4vci/OpenId4VciIssuanceRequest.kt

```diff
@@ -181,4 +185,34 @@ internal class OpenId4VciIssuanceRequest(
     private fun getDidAndKeyIdFromHeader(kid: String): Pair<String?, String> {
         return JwaCryptoHelper.extractDidAndKeyId(kid)
     }
+
+    private suspend fun sendIssuanceCallbackIfRequestStateAndCallbackExist(result: VerifiedIdResult<VerifiedId>) {
+        val requestState = credentialOffer.issuer_session
+        val issuanceCallbackUrl = credentialMetadata.notificationEndpoint ?: return
+
+        var issuanceCompletionCode: IssuanceCompletionResponse.IssuanceCompletionCode =
+            IssuanceCompletionResponse.IssuanceCompletionCode.ISSUANCE_FAILED
+        var issuanceCompletionErrorDetails: IssuanceCompletionResponse.IssuanceCompletionErrorDetails? =
+            null
+        result.fold(
+            onSuccess = {
+                issuanceCompletionCode =
+                    IssuanceCompletionResponse.IssuanceCompletionCode.ISSUANCE_SUCCESSFUL
+            },
+            onFailure = {
+                issuanceCompletionErrorDetails = when (it) {
+                    is VerifiedIdException -> IssuanceCompletionResponse.IssuanceCompletionErrorDetails.ISSUANCE_SERVICE_ERROR
```

Comment:
This covers the failure when response is sent to service. There would also scenarios like user canceled and while fetching the request. Callback needs to be sent in those scenarios also.

Reply:
Implemented the former. The latter is not possible since the notification endpoint is contained in the issuer metadata itself.